eng/scripts/Language-Settings.ps1

```diff
@@ -24,7 +24,12 @@ function Get-javascript-PackageInfoFromRepo ($pkgPath, $serviceDirectory)
     $jsStylePkgName = $projectJson.name.Replace("@", "").Replace("/", "-")
 
     $pkgProp = [PackageProps]::new($projectJson.name, $projectJson.version, $pkgPath, $serviceDirectory)
-    $pkgProp.SdkType = $projectJson.psobject.properties['sdk-type'].value
+    if ($projectJson.psobject.properties.name -contains 'sdk-type') {
```

Comment:
should it have `.name`? I'll check to make sure.

Reply:
Never mind this is correct.

Reply:
Yes that is essentially selecting the name property for our properties so we can search just the names.